### PR TITLE
Redlining tools panel

### PR DESCRIPTION
**Pull request by annarieger**
This PR introduces a redlining tools functionality. The following tools are supported at the moment:
- Draw point
- Draw linestring
- Draw polygon
- Add post-it
- Move drawn object
- Copy/clone drawn object
- Modify drawn object
- Delete drawn object
- Styler for point/line/polygon features ("simple" styling with stroke color/width, point radius and fillcolor properties, no images as sources for point features are supported now)

The implementation is based on https://github.com/terrestris/BasiGX/blob/master/src/view/container/RedlineStyler.js and  https://github.com/terrestris/BasiGX/blob/master/src/view/container/Redlining.js classes of `BasiGX` library. It was significantly adapted to match the MVVM structure.

Corresponding PR in backend: https://github.com/terrestris/momo3-backend/pull/30

Please review @buehner @marcjansen

### CODE REVIEW

File: app/view/button/ShowRedliningToolsPanelController.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                    stroke: new ol.style.Stroke({
+                        color: '#ececec',
+                        width: 2
+                    }),
+                    fill: new ol.style.Fill({
+                        color: '#5fa2dd'
+                    })
+                })
+            }
+        );
```
Comment (buehner) [inline app/view/button/ShowRedliningToolsPanelController.js:122]: Is it possible to move/externalize the config of the `btnPanel` in some way? Maybe as default values in the `RedliningToolsPanel` view? Or is it even possible to make clever use of data binding here, e.g. for the color values!?

If this is too much for know, just ignore this comment (but keep it in mind :wink: )

**Comment by buehner** [comment]
As this PR is large, i just had a quick look at the code and did not find any major issues. The code looks clean and well structured. So feel free to merge, if CI remains happy :wink:

**Comment by annarieger** [comment]
Thx for review @buehner! :pineapple: 

I've moved the default style definitions to the view model in 8b8f773 as suggested. Please check if it's ok and merge at will.

**Comment by buehner** [comment]
:+1:

**Pull request merged**